target/linux/ar71xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -138,6 +138,8 @@ case "$FIRMWARE" in
 	archer-c59-v1|\
 	archer-c60-v1)
 		ath10kcal_extract "art" 20480 12064
+		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
```

Comment:
@chunkeey Now i send patch to ath10k-devel http://lists.infradead.org/pipermail/ath10k/2017-July/009930.html
ps. This is my first patch using mail list :)

@pepe2k i try fix @blogic qca9888 commit. 
If board-2.bin is not be fixed with boarddata0.bin then better way to use symlink or again download ART from flash?